Source/HelixToolkit.Wpf/Importers/ObjReader.cs

```diff
@@ -52,6 +52,7 @@ public ObjReader(Dispatcher dispatcher = null)
             : base(dispatcher)
         {
             this.IgnoreErrors = false;
+            this.SwitchYZ = true;

```

Comment:
This is a breaking change. I think it would be better to set the default value to `false`, then all the changes to the test files can be reverted!